layer/projects/progress_tracker_updater.py

```diff
@@ -115,12 +115,26 @@ def _update_tracker(self, task: PBTask) -> None:
     def _handle_task_succeeded(self, task: PBTask) -> None:
         task_id = task.id
         task_type = task.type
+        project_full_name = get_current_project_full_name()
         if task_type == PBTask.TYPE_DATASET_BUILD:
             dataset_name = task_id
-            dataset_path = f"{get_current_project_name()}/datasets/{dataset_name}"
```

Comment:
this is coming from flow-manager `DatasetBuildActivity` which rebuilds a path to a dataset name based on context